examples/action_dann/main.py

```diff
@@ -13,7 +13,7 @@
 from pytorch_lightning.callbacks import LearningRateMonitor, ModelCheckpoint, TQDMProgressBar
 
 from kale.loaddata.video_access import VideoDataset
-from kale.loaddata.video_multi_domain import VideoMultiDomainDatasets
+from kale.loaddata.video_multi_domain import VideoBinaryDomainDatasets
```

Comment:
The name `VideoBinaryDomainDatasets` may be misleading, as `binary` can refer to binary video action classification. Would `VideoTwoDomainDatasets` or `VideoBiDomainDatasets` be clearer?